ticdc/ticdc-faq.md

```diff
@@ -132,15 +132,15 @@ For more information, refer to [Create a replication task](/ticdc/manage-ticdc.m
     * Increase the `replica.fetch.max.bytes` value in `server.properties` to `1073741824` (1 GB).
     * Increase the `fetch.message.max.bytes` value in `consumer.properties` to make it larger than the `message.max.bytes` value.
 
-## When TiCDC replicates data to Kafka, does it write all the changes in a transaction into one message? If not, on what basis does it divide the changes?
+## When TiCDC replicates data to Kafka, can I control the maximum size of a single message in TiDB?
 
-No. According to the different distribution strategies configured, TiCDC divides the changes on different bases, including `default`, `row id`, `table`, and `ts`.
+When `protocol` is set to `avro` or `canal-json`, messages are sent per row change. A single Kafka message contains only one row change and is generally no larger than Kafka's limit. Therefore, there is no need to limit the size of a single message. If the size of a single Kafka message does exceed Kakfa's limit, refer to [Why does the latency from TiCDC to Kafka become higher and higher?](/ticdc/ticdc-faq.md#why-does-the-latency-from-ticdc-to-kafka-become-higher-and-higher).
 
-For more information, refer to [Replication task configuration file](/ticdc/manage-ticdc.md#task-configuration-file).
+When `protocol` is set to `open-protocol`, messages are sent in batches. Therefore, one Kafka message might be excessively large. To avoid this situation, you can configure the `max-message-bytes` parameter to control the maximum size of data sent to the Kafka broker each time (optional, `10MB` by default). You can also configure the `max-batch-size` parameter to specify the maximum number of change records in each Kafka message. Currently, the setting only takes effect when Kafka's `protocol` is `open-protocol` (optional, `16` by default).
```

Comment:
"Currently, the setting only takes effect when Kafka's `protocol` is `open-protocol`" this sentence is not in the Chinese version. Why?

Reply:
You are right. The English is not consistent with the Chinese. I have fixed it in [21288f4](https://github.com/pingcap/docs/pull/10843/commits/21288f4eb69b9708929e7f69320f961e88d7fd94).